test/plugin/test_out_google_cloud.rb

```diff
@@ -527,6 +562,19 @@ def test_one_log
     verify_log_entries(1, COMPUTE_PARAMS)
   end
 
+  def test_one_log_with_grpc_on
+    setup_gce_metadata_stubs
+    service = setup_grpc_logging_stubs
+
+    d = create_driver_with_grpc_mock(USE_GRPC_CONFIG)
+    d.emit('message' => log_entry(0))
+    d.run
+
+    @requests_sent = service.requests_received
```

Comment:
Question:

I saw we use an instance variable "logs_sent" in other tests, so I made "requests_sent" an instance variable as well. Yet is there an known reason why we are doing this instead of using a local variable and pass it to the verify_grpc_log_entries method as a parameter? Here since tests are not running in parallel, it probably does not matter. Just curious if this is some convention with us or with Ruby.